DOCS-14165: SDS Logs Findings explorer (#37797)
DataDog/documentation: content/en/security/sensitive_data_scanner/guide/investigate_sensitive_data_findings.md

The Sensitive Data Scanner's [Findings][1] page categorizes and prioritizes sensitive data findings so that you can investigate, collaborate, and document your findings, and answer those questions.

> {{< img src="sensitive_data_scanner/sds_logs_findings_explorer.png" alt="Sensitive Data Scanner Logs Findings explorer grouped by service, with a service expanded to show scanner findings, match counts, and trend charts." style="width:100%;" >}}

Reviewer: i'd say to reuse the same screenshot as: https://github.com/DataDog/documentation/pull/37797/changes#r3623483558

this would mean explaining the "Go back ->" button, because we show the new explorer by default if the customer has at least one logs finding

> {{< img src="sensitive_data_scanner/sds_logs_findings_explorer.png" alt="Sensitive Data Scanner Logs Findings explorer grouped by service, with the auth-dotnet service expanded to show critical credit card scanner findings, match counts, trend charts, and last detected times." style="width:100%;" >}}

Reviewer 2: A bit long, took a stab at trimming. Take or leave.

```suggestion
{{< img src="sensitive_data_scanner/sds_logs_findings_explorer.png" alt="Sensitive Data Scanner Logs Findings explorer grouped by service, with a service expanded to show scanner findings, match counts, and trend charts." style="width:100%;" >}}
```

## Triage sensitive data findings

Navigate to the [Findings][1] page to see all sensitive data findings within the selected time frame and start investigating them.

{{< tabs >}}
{{% tab "Logs" %}}

> The Logs Findings explorer is an updated experience for investigating log findings. To access it, click {{< ui >}}Try the new version{{< /ui >}} in the blue banner at the top of the Telemetry tab. APM, RUM, and Events findings are not available in this explorer. To view those findings, click {{< ui >}}Go back{{< /ui >}} in the banner.

Reviewer 2: Just noting that you reference the UI tab as **Telemetry** here but **Sensitive Data Rule Findings** for "APM, RUM, and Events" and "Cloud Storage" tabs on the docs page. Is that intentional, or worth updating for consistency?

Author: Good catch. It was intentional. We're starting with a new Logs explorer, but later we'll have separate explorers for APM and RUM, as well. Technically, logs are a type of telemetry, so the PM and I agreed to name the second tab "APM, RUM, and Events" instead of "Telemetry".

To investigate a log finding:

1. Use {{< ui >}}Group by{{< /ui >}} to organize findings by {{< ui >}}Rule{{< /ui >}}, {{< ui >}}Logs Pattern{{< /ui >}}, or {{< ui >}}Service{{< /ui >}}. To surface findings where sensitive data is actively exposed, filter by {{< ui >}}Leaking{{< /ui >}} in the {{< ui >}}Match State{{< /ui >}} facet.
2. Click a finding to open the detail panel.
3. At the top of the panel, check {{< ui >}}First Detected{{< /ui >}} and {{< ui >}}Last Detected{{< /ui >}} to understand how long the exposure has been active.
4. In the summary section, review {{< ui >}}Match State{{< /ui >}}, {{< ui >}}Service{{< /ui >}}, {{< ui >}}Environment{{< /ui >}}, and {{< ui >}}Total matches{{< /ui >}} to understand the scope of the exposure.
5. Review the {{< ui >}}Logs Pattern{{< /ui >}} to understand the format of the log line where sensitive data was detected.

> 6. In the {{< ui >}}Example Logs{{< /ui >}} section, review representative examples of affected logs. Click {{< ui >}}Show log{{< /ui >}} to expand an example and inspect its log message, fields, and attributes inline.

Reviewer: In the {{< ui >}}Example Logs{{< /ui >}} section, review up to 5 representative examples of affected logs. When an example log expires it is replaced with the next matching event. Click {{< ui >}}Show log{{< /ui >}}  to expand an example and inspect its log message, fields, and attributes inline. By default, example logs are stored for 3 days and are accessible to all users with the Data Scanner Read permission. If you want to store these representative logs for a different period, contact [Support](https://docs.datadoghq.com/help/).

^ this comes from legal so we need to add it in

7. Review {{< ui >}}Matches Trend{{< /ui >}} to see how match volume has changed over the past week. Use {{< ui >}}Related Access and Configuration Events{{< /ui >}} to check whether recent access events or changes to the scanning group or scanning rule line up with changes in match volume.

> Additionally, you can also:

Reviewer 2: ```suggestion
Additionally, you can:
```
